src/cdk/drag-drop/directives/drop-list.ts

```diff
@@ -125,7 +125,13 @@ export class CdkDropList<T = any> implements OnDestroy {
 
   /** Whether sorting within this drop list is disabled. */
   @Input('cdkDropListSortingDisabled')
-  sortingDisabled: boolean;
+  get sortingDisabled(): boolean {
+    return this._sortingDisabled;
+  }
+  set sortingDisabled(value: boolean) {
+    this._sortingDisabled = coerceBooleanProperty(value);
+  }
+  private _sortingDisabled: boolean;
```

Comment:
All of the changes in this file aren't necessary, because the coercion happens inside the `beforeStarted` subscription.